### ISSUE TITLE
pypodman mount: option `--notruncate` is not implemented

### ISSUE THREAD

**Issue by Edward5hen** [containers/libpod#1961]
**Is this a BUG REPORT or FEATURE REQUEST?**:

**kind feature**

**Description**
As title, mount option `--notruncate` is not implemented by pypodman:
https://github.com/containers/libpod/blob/1f547b2936af06656eaf579c3adaf8f345d227fc/contrib/python/pypodman/pypodman/lib/actions/mount_action.py#L12-L25
For keeping consistency with podman, pypodman should implement option `--notruncate` as well.

**Comment by rhatdan** [comment]
Well this is worse that that, it is not actually showing what is mounted, but it is hard coded to list the merged directories.  We need this in the formal API.

**Comment by jwhonce** [comment]
@Edward5hen pypodman mount --no-trunc <id> works.  The call to super() pulls in --no-trunc

**Comment by jwhonce** [comment]
@rhatdan When I run podman mount on a container I get the only the merged directory. I don't know if that is the container I'm picking or a change you are looking for in the python and go CLI's.

**Comment by Edward5hen** [comment]
@jwhonce Yes, you're right. Sorry I didn't notice that, but the option for podman is `--notruncate` here we have `--no-trunc|--no-truncate`, I think pypodman should have the same behavior, or users might be confused about this.
https://github.com/containers/libpod/blob/master/docs/podman-mount.1.md#options
> --notruncate
> Do not truncate IDs in output.

**Comment by baude** [comment]
I have evaluated this and this really needs @jwhonce's hands.  He has designed a customer add parent arg parser where he adds `no-` in front of options.  therefore i think he needs to decide how to handle this.  on that note @jwhonce , the description for no-truncate is inaccurate as it describes trunc and not no-trunc.
```
--no-truncate         Truncate id's and other long fields
```
